tests/test_workers/test_tasks/test_utils.py

```diff
@@ -106,6 +106,73 @@ def test_set_registry_token(
     mock_rdc.assert_called_once_with()
 
 
+@pytest.mark.parametrize('config_exists', (True, False))
+@pytest.mark.parametrize('template_exists', (True, False))
+@mock.patch('os.path.expanduser')
+@mock.patch('os.remove')
+@mock.patch('os.path.exists')
+@mock.patch('iib.workers.tasks.utils.open')
+@mock.patch('iib.workers.tasks.utils.json.dump')
+@mock.patch('iib.workers.tasks.utils.reset_docker_config')
+def test_set_registry_auths(
```

Comment:
Let's add a test where we validate that the auth info from `registry_auths` overwrites whatever is in the config.

Reply:
I added 1 extra registry which should be overwritten with provided registry_auths. Does it look sufficient? https://github.com/release-engineering/iib/pull/227/files#diff-ad30f2cc32db90c116232cb76cc707702b22936e22f90e0fd5f5d215aa984ec9R134